dll/win32/netcfgx/tcpipconf_notify.c

```diff
@@ -2144,10 +2221,53 @@ TcpipAltConfDlg(
     WPARAM wParam,
     LPARAM lParam)
 {
-    switch(uMsg)
+    TcpipConfNotifyImpl *This;
+    LPPROPSHEETPAGE page;
+    BOOL bEnabled;
+
+    switch (uMsg)
     {
         case WM_INITDIALOG:
+        {
+            page = (LPPROPSHEETPAGE)lParam;
+            This = (TcpipConfNotifyImpl*)page->lParam;
+            SetWindowLongPtr(hwndDlg, DWLP_USER, (LONG_PTR)This);
+            if (This->pCurrentConfig)
+            {
+                InitializeTcpipAltDlgCtrls(hwndDlg, This->pCurrentConfig);
+            }
```

Comment:
```suggestion
                InitializeTcpipAltDlgCtrls(hwndDlg, This->pCurrentConfig);
```
just a suggestion, no must. I know that it's a matter of taste.